docs.md

```diff
@@ -42,24 +42,24 @@ clone:
 | `recursive`               | `false`                             | Clones submodules recursively                                                                                                           |
-| `skip_verify`             | `false`                             | Skips the SSL verification                                                                                                              |
+| `skip-verify`             | `false`                             | Skips the SSL verification                                                                                                              |
 | `tags`                    | `false` (except on tag event)       | Fetches tags when set to true, default is false if event is not tag else true                                                           |
-| `submodule_overrides`     | _none_                              | Override submodule urls                                                                                                                 |
-| `submodule_update_remote` | `false`                             | Pass the --remote flag to git submodule update                                                                                          |
-| `submodule_partial`       | `true`                              | Update submodules via partial clone (depth=1)                                                                                           |
-| `custom_ssl_path`         | _none_                              | Set path to custom cert                                                                                                                 |
-| `custom_ssl_url`          | _none_                              | Set url to custom cert                                                                                                                  |
+| `submodule-overrides`     | _none_                              | Override submodule urls                                                                                                                 |
+| `submodule-update-remote` | `false`                             | Pass the --remote flag to git submodule update                                                                                          |
+| `submodule-partial`       | `true`                              | Update submodules via partial clone (depth=1)                                                                                           |
+| `custom-ssl-path`         | _none_                              | Set path to custom cert                                                                                                                 |
+| `custom-ssl-url`          | _none_                              | Set url to custom cert                                                                                                                  |
```

Comment:
I couldn't find anything about these two options in the codebase, do they actually exist?